actions.go

```diff
@@ -19,7 +19,13 @@ var prohibitedSchemaKeys = map[string]bool{"$ref": true, "$schema": true}
 
 var actionNameRule = regexp.MustCompile("^[a-z0-9](?:[a-z0-9-]*[a-z0-9])?$")
 
-// Actions defines the available actions for the charm.  Additional params
+// Export `actionNameRule` variable to different contexts. The best practice
```

Comment:
probably don't need to explain that it's a best practice in the code comments

Reply:
Updated